### ISSUE TITLE
A11y_AzureDataStudio_Welcome_AzureDataStudio-Insiders_ScreenReader : Screen reader users are unable to access the expanded content of the "New" control also list structure is not provided.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/azuredatastudio#11545]
**"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”**
GitHubTags:#A11y_AzureDataStudioJuly2020;#A11y_July27_2020_TestPass;#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#MAC;#Screenreader;#VoiceOver;#A11ySev1;#Benchmark;#MAS1.3.1;#MAS2.1.1#FTP;

### Environment Details:

Application Name: Azure Data Studio
Application Version: 1.21.0-insider
Commit: eccf3cf
Date: 2020-07-24T09:28:31.172Z
VS Code: 1.48.0
Electron: 9.1.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina (Version 10.15.6 (19G73)
Screen Reader: VoiceOver
MAS References: MAS1.3.1, MAS2.1.1

### Repro Steps:

1. Launch Azure Data Studio Insiders application.
2. If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3. Start screen reader, navigate to "New" control and activate it then observed the screen reader users are able to access the expanded content.
4. Also, observe if the list structure is provided for the expanded options.

### Actual:
**Issue 1:**
When screen reader users navigate to the "New" control and activate it, then the users try to access the expanded options using the arrow keys but they were unable to access the options using keyboard which is incorrect.

**Issue 2:**
There is no list structured provided for the expanded content which appears on activating the "New" control for screen reader users which is incorrect as the users will not be able to know the number of expanded option.

### Expected:
**Issue 1:**
The screen reader users should be able to access the expanded options using the arrow keys of the "New" control.

**Issue 2:**
Proper list structure should be provided for the expanded options of the "New" control for screen reader users.

### User Impact:
**Issue 1:**
If the screen reader users are unable to access the interactive element using keyboard then users will miss out on the information that is available for other users.

**Issue 2:**
If proper structure is not provided for the content present on the screen then the screen reader users will have difficulty in understanding the content present on the screen.

### Attachment link for Reference:
[11545_A11y_AzureDataStudio_Welcome_AzureDataStudio-Insiders_ScreenReader Screen reader users are unable to access the expanded content of the "New" control also list structure is not provided.zip](https://github.com/microsoft/azuredatastudio/files/4985603/11545_A11y_AzureDataStudio_Welcome_AzureDataStudio-Insiders_ScreenReader.Screen.reader.users.are.unable.to.access.the.expanded.content.of.the.New.control.also.list.structure.is.not.provided.zip)

**Comment by Raghu-addepalli** [comment]
Issue is fixed Hence closing the bug and added attachment.
Githubtags:  #Closed;
[11545_Fixed.zip](https://github.com/microsoft/azuredatastudio/files/10813025/11545_Fixed.zip)
